pypeit/calibrations.py

```diff
@@ -507,20 +508,30 @@ def get_flats(self):
 
         # TODO -- Allow for separate pixelflat and illumflat images
         # Generate the image
-        stacked_flat = None
-        if len(illum_image_files) > 0:
-            stacked_flat = buildimage.buildimage_fromlist(self.spectrograph, self.det,
-                                                          self.par['illumflatframe'],
-                                                          illum_image_files, dark=self.msdark,
-                                                          bias=self.msbias, bpm=self.msbpm)
-        if stacked_flat is None and len(pixflat_image_files) > 0:
-            stacked_flat = buildimage.buildimage_fromlist(self.spectrograph, self.det,
+        illum_flat, pix_flat = None, None
+        # Check if the image files are the same
+        pix_is_illum = Counter(illum_image_files) == Counter(pixflat_image_files)
```

Comment:
maybe use `np.array_equal` instead?

Reply:
Both `illum_image_files` and `pixflat_image_files` are of type list, so `Counter` is the best option here to make sure that the lists have identical contents (regardless of order).